foundation/foundation-upgrade/src/test/java/com/boozallen/aissemble/upgrade/migration/v1_7_0/DataLineageMigrationSteps.java

```diff
@@ -0,0 +1,51 @@
+package com.boozallen.aissemble.upgrade.migration.v1_7_0;
+
+/*-
+ * #%L
+ * aiSSEMBLE::Foundation::Upgrade
+ * %%
+ * Copyright (C) 2021 Booz Allen
+ * %%
+ * This software package is licensed under the Booz Allen Public License. All Rights Reserved.
+ * #L%
+ */
+
+import com.boozallen.aissemble.upgrade.migration.AbstractAissembleMigration;
+import com.boozallen.aissemble.upgrade.migration.AbstractMigrationTest;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+
+import static junit.framework.Assert.assertFalse;
+import static junit.framework.TestCase.assertTrue;
+
+public class DataLineageMigrationSteps extends AbstractMigrationTest {
+    private AbstractAissembleMigration migration;
+
+    @Given("an existing downstream project with data lineage is less than VERSION_AISSEMBLE_one_seven_zero")
+    public void anExistingDownstreamProjectWithDataLineageIsLessThanVERSION_AISSEMBLE_one_seven_zero() {
+        testFile = getTestFile("v1_7_0/DataLineageMigration/migration/CdiContainerFactory_1_6_0.java");
+        migration = new DataLineageAiopsReferencesMigration();
+    }
+
+    @Given("an existing downstream project with correct data package lineage imports are present")
+    public void anExistingDownstreamProjectWithCorrectDataPackageLineageImportsArePresent() {
+        testFile = getTestFile("v1_7_0/DataLineageMigration/skip-migration/CdiContainerFactory_Correct_Package_imports.java");
+        migration = new DataLineageAiopsReferencesMigration();
+    }
+
+    @When("the data lineage version migration executes")
+    public void theDataLineageVersionMigrationExecutes() {
+        performMigration(migration);
+    }
+
+    @Then("the imports are updated in the data lineage modules with the new package name")
+    public void theImportsAreUpdatedInTheDataLineageModulesWithTheNewPackageName() {
+        assertTrue("Data Lineage modules are references an older package in imports.", shouldExecute);
```

Comment:
S: This does _not_ validate the core assertion of the `then`.  Firstly, I think it's fine to check that `shouldExecute` was true, but the more important flag is the one that says the file was actually migrated.  Secondly, we need to verify that the old package references were actually updated.  There are two approaches we tend to take and I think both are valid (we may need to just make a team norm): either create a "validation" file where you manually make the changes you'd expect the migration to make and validate that the migrated file matches the validation file, or you programmatically read in and check the migrated file with specific logic.